app/src/main/java/com/mobiledevelopmentfoodapp/dto/Order.kt

```diff
@@ -1,10 +1,14 @@
 package com.mobiledevelopmentfoodapp.dto
 
-//import androidx.room.Entity
-//import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 
-data class Order(var customerId: Int, var name: String, var products: ArrayList<Food>, var totalPrice: Double, @SerializedName("id") var orderId:Int = 0 ) {
+data class Order(
```

Comment:
Formatting and visibility modifiers. Visibility modifier best practices can be found at https://kotlinlang.org/docs/visibility-modifiers.html Formatting can be found at https://kotlinlang.org/docs/coding-conventions.html#formatting